crates/cxx-qt-gen/src/naming/type_names.rs

```diff
@@ -801,6 +801,29 @@ mod tests {
             .is_ok());
     }
 
+    #[test]
+    fn test_shared_type() {
+        let items = [parse_quote! {
+            mod ffi {
+             enum MyEnum {
+                Yes,
+                No
+            }
+            extern "C++" {
+                type MyEnum;
+            }
+            }
+        }];
+        let mut types = TypeNames::default();
+        // assert!(types
```

Comment:
assume this test isn't finished yet :-)

Reply:
I think it works, just wanted to be sure. removed the commented code